README.md

```diff
@@ -0,0 +1,28 @@
+# app-elfloader
```

Comment:
I added a sentence and a reference to the Run App ELF loader repo, which is more elaborate to avoid duplication. Or would you rather have me copy some more information from there?